plugins-structure/apps/politeia/src/pages/Home/homeSlice.js

```diff
@@ -0,0 +1,113 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import { getTokensToFetch } from "@politeiagui/core/records/utils";
+import { validateRecordsPageSize } from "@politeiagui/core/records/validation";
+import { ticketvoteSummaries } from "@politeiagui/ticketvote/summaries";
+import { getHumanReadableTicketvoteStatus } from "@politeiagui/ticketvote/utils";
+import {
+  validateTicketvoteStatus,
+  validateTicketvoteSummariesPageSize,
+} from "@politeiagui/ticketvote/validation";
+import { records } from "@politeiagui/core/records";
+import {
+  validateInventoryIsLoaded,
+  validateInventoryListLength,
+} from "./validation";
+
+const initialStateStatus = {
+  lastTokenPos: null,
+  status: "idle",
+  error: null,
+};
+
+export const initialState = {
+  unauthorized: initialStateStatus,
+  authorized: initialStateStatus,
+  started: initialStateStatus,
+  finished: initialStateStatus,
+  approved: initialStateStatus,
+  rejected: initialStateStatus,
+  ineligible: initialStateStatus,
+};
+
+const piFilenames = ["proposalmetadata.json", "votemetadata.json"];
+
+// Thunks
+export const fetchNextBatch = createAsyncThunk(
+  "home/fetchNextBatch",
+  async (status, { dispatch, rejectWithValue, getState }) => {
+    try {
+      const {
+        ticketvoteInventory,
+        records: recordsState,
+        recordsPolicy,
+        ticketvotePolicy,
+        home,
+      } = getState();
+
+      const summariesPageSize = ticketvotePolicy.policy.summariespagesize;
+      const recordsPageSize = recordsPolicy.policy.recordspagesize;
+      const { tokens, last } = getTokensToFetch({
+        records: recordsState.records,
+        pageSize: Math.min(summariesPageSize, recordsPageSize),
+        inventoryList: ticketvoteInventory[status].tokens,
+        lastTokenPos: home[status].lastTokenPos,
```

Comment:
we should convert the status to a readable status using the `getHumanReadableTicketvoteStatus` from ticketvote utils